source/docs/articles/drupal/adding-yml-file.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Adding a services.yml File
+description: Learn how to add a services.yml file for Drupal 8.
+category:
+  - developing
+  - drupal
+keywords: drupal, services.yml, yml file
+---
+Drupal 8 includes the `sites/default/default.services.yml` configuration file to define services used, eliminating the need to hack core to swap functionality. To create and/or modify this file, you'll need to tweak some permissions, otherwise you will receive a **Permission denied** error.
+
+<div class="alert alert-danger">
+<h4>Warning</h4>
+Due to permission handling issues observed when using Git, we recommend creating the <code>services.yml</code> file via SFTP. For instructions, see <a href="/docs/articles/sites/code/developing-directly-with-sftp-mode">Developing on Pantheon Directly with SFTP Mode</a>.
```

Comment:
I don't know that you need to make this warning so restrictive; if someone was already in the middle of making modifications in git mode, it would be inconvenient for them to switch to SFTP mode just to make changes to this one file.  It would be better, I think, not make note that the services.yml file should be protected, and it is not possible to set these permission bits with git, so git users should switch to sftp mode briefly, once they are done, to change the permissions.